shared/cfg/guest-os/Linux.cfg

```diff
@@ -27,11 +27,9 @@
         time_command = date +'TIME: %a %m/%d/%Y %H:%M:%S.%N'
         time_filter_re = "(?:TIME: \w\w\w )(.{19})(?:\.\d\d)"
         time_format = "%m/%d/%Y %H:%M:%S"
-        guest_load_command = "dd if=/dev/urandom of=/dev/null"
-        guest_load_instances = 2
-        guest_load_stop_command = "killall -9 dd"
-        host_load_command = "bzip2 -c --best /dev/urandom > /dev/null"
-        host_load_instances = 8
+        guest_load_command = touch /tmp/guest_load_timedrift; taskset -c %s /bin/bash -c 'while [ -e /tmp/guest_load_timedrift ]; do X=1; done&'
+        guest_load_stop_command = rm -f /tmp/guest_load_timedrift
+        host_load_command = /bin/bash -c 'for ((;;)); do X=1; done'
```

Comment:
please wrap all the commands in double quotes.

Reply:
@balamuruhans Thanks, updated it.